tests/test_tasker.py

```diff
@@ -40,12 +40,12 @@
 def setup_module(module):
     if MOCK:
         return
-    d = docker.Client()
+    d = docker.Client()  # TODO: current version of python-docker does not have Client anymore
```

Comment:
IMHO this needs investigation, we may need to freeze python-docker version in requirements.txt

Reply:
Should we try `APIClient` and fall back to `Client`, like we've done elsewhere?